content/blog/2020/06/camel-k-release-1.0.0/index.md

```diff
@@ -0,0 +1,250 @@
+
+The Camel K embedded auto-scaling feature works really well when you want to react to some Knative events:
+
+*listener.groovy*
+```groovy
+from('knative:event/device-event')
+  .to('http://myhost/webhook/random-id')
+``` 
+
+This integration is configured to receive all events with `type=device-event` and **scales automatically with the load** because it is materialized into a [Knative Serving Service](https://knative.dev/docs/serving/spec/knative-api-specification-1.0/#service)
+and automatically [subscribed to the Eventing Broker via a Trigger](https://knative.dev/docs/eventing/broker/).
+
+It then receives a [CloudEvent](https://cloudevents.io/) when your IoT devices produce something and scales down to zero if there's no data coming.
+You just need to create it (as before, just `kamel run listener.groovy`), all the remaining configuration is 
+done **dynamically by the Camel K operator**.
+
+We've added much more features for having a better integration with the Knative ecosystem and we've also fixed some compatibility and performance issues that were present in previous versions. The user experience is now much smoother.
+
+If you are a Knative YAML developer (!), instead of using Camel K directly, you also have the option to use **[Knative Camel Sources](https://knative.dev/docs/eventing/samples/apache-camel-source/)** which are part of the Knative release. They are wrappers for Camel K integrations that are compatible with all the tools used by Knative developers (such as the `kn` CLI or the OpenShift serverless console).
+Sources in Knative can only push data into the various Knative endpoints, but not the other way around (i.e. they cannot be used to publish data from Knative to the outside).
+In Camel K you don't have this distinction: the Route is the fundamental building block of a Camel integration and you can do whatever you want with it. 
+
+## Fast startup and low memory
+
+We cannot say we're serverless without mentioning the work that we've been doing in improving the performance of Camel K integrations.
+
+Starting from **Camel 3.3.0** which is the default version used by Camel K 1.0.0, you can benefit from all improvements that have been made directly in the core to make it much more lightweight. Claus Ibsen has written a series of blog posts ([part 1](http://www.davsclaus.com/2020/01/apache-camel-31-more-camel-core.html?m=1), [part 2](http://www.davsclaus.com/2020/01/apache-camel-31-more-camel-core_30.html?m=1), [part 3](http://www.davsclaus.com/2020/02/apache-camel-31-more-camel-core.html?m=1), [part 4](http://www.davsclaus.com/2020/03/apache-camel-32-reflection-free.html?m=1)) to highlight what has been changed in the 3.x Camel timeline to reduce memory footprint and speedup the startup time, which is foundamental when running integrations in a serverless environment.
```

Comment:
the core -> the Camel core

Reply:
Claus Ibsen has written ... ->

More in depth details of the Camel core improvements can be found the following blog series